src/app/layout.js

```
@@ -15,25 +19,37 @@ const robotoMono = Roboto_Mono({
-        <NavBar />
+      <body className={`${roboto.variable} ${robotoMono.variable} antialiased`} >
+        { !isDashboard && <NavBar />}
         <section>
           {children}
         </section>
-        <Footer />
+        { !isDashboard && <Footer />}
+
+
+        {!isDashboard ?
+        <button onClick={() => window.location.href = "donation"} className='red_color don_icon_animate' title='Faire un DON'>
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)